### ISSUE TITLE
analyze_gyration_tensor test broken on aarch64

### ISSUE THREAD

**Issue by junghans** [espressomd/espresso#2469]
Building 4.0.1 on openSUSE, I found:
```
[ 3485s] 105/113 Test #105: analyze_gyration_tensor .................***Failed    3.39 sec
[ 3485s] Features:  ['BOND_ANGLE', 'BUCKINGHAM', 'COLLISION_DETECTION', 'ELECTROSTATICS', 'EXCLUSIONS', 'EXTERNAL_FORCES', 'FFTW', 'GAUSSIAN', 'GHOSTS_HAVE_BONDS', 'GSL', 'HERTZIAN', 'LANGEVIN_PER_PARTICLE', 'LATTICE', 'LB', 'LB_BOUNDARIES', 'LENNARD_JONES', 'LENNARD_JONES_GENERIC', 'LJCOS', 'LJCOS2', 'MASS', 'MORSE', 'NPT', 'P3M', 'PARTIAL_PERIODIC', 'SOFT_SPHERE', 'TABULATED']
[ 3485s] F..
[ 3485s] ======================================================================
[ 3485s] FAIL: test_gyration_tensor (__main__.AnalyzeGyration)
[ 3485s] ----------------------------------------------------------------------
[ 3485s] Traceback (most recent call last):
[ 3485s]   File "/home/abuild/rpmbuild/BUILD/espresso-4.0.1/build/testsuite/python/analyze_gyration_tensor.py", line 68, in test_gyration_tensor
[ 3485s]     np.allclose(np.abs(res['eva0'][1]), [0., 0., 1.], atol=1e-6))
[ 3485s] AssertionError: False is not true
[ 3485s] 
[ 3485s] ----------------------------------------------------------------------
[ 3485s] Ran 3 tests in 0.390s
[ 3485s] 
[ 3485s] FAILED (failures=1)
[ 3485s] -------------------------------------------------------
[ 3485s] Primary job  terminated normally, but 1 process returned
[ 3485s] a non-zero exit code.. Per user-direction, the job has been aborted.
[ 3485s] -------------------------------------------------------
[ 3485s] --------------------------------------------------------------------------
[ 3485s] mpiexec detected that one or more processes exited with non-zero status, thus causing
[ 3485s] the job to be terminated. The first process to do so was:
[ 3485s] 
[ 3485s]   Process name: [[2320,1],0]
[ 3485s]   Exit code:    1
[ 3485s] --------------------------------------------------------------------------
[ 3485s] 
```

**Comment by junghans** [comment]
I cannot reproduce this on Fedora, so it must be an openSUSE thing.

**Comment by junghans** [comment]
Everything fixed in F30 with boost-1.69.0!
